LICENSE

```diff
@@ -200,3 +200,30 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
+----
+
+This product bundles Glyphicon Halflings ( http://glyphicons.com/
+ ) via Bootstrap 3's distributing these under The MIT License.
+
+This product bundles angular-ui / ui-sortable / sortable.js ( https://github.com/angular-ui/ui-sortable/blob/master/src/sortable.js ) under The MIT License.
+
+
+The MIT License (MIT)
```

Comment:
Would a link to https://opensource.org/licenses/MIT
Or reference links like
1. node_modules/bootstrap/LICENSE
2. node_modules/font-awesome/LICENSE
3. node_modules/normalize.less/LICENSE

work?